FileSystem/file_mapper_tests.js

```diff
@@ -97,16 +101,14 @@ function file_folder_existence_tests() {
 }
 
 function file_folder_delete_tests() {
-  QUnit.test('deleteFile - Delete a file testing', function() {
-    expect(2);
+  QUnit.test('deleteFile - Delete a file testing', 2, function() {
     var file1 = 'c:\\User\\Desktop\\folder1\\deletethis.txt';
     ok(FileMapper.hasFile(file1), 'File exists');
     FileMapper.deleteFile(file1);
     ok(!FileMapper.hasFile(file1), 'File does not exist');
   });
 
-  QUnit.test('deleteFolder - Delete a folder testing', function() {
-    expect(2);
+  QUnit.test('deleteFolder - Delete a folder testing', 2, function() {
```

Comment:
Should we be passing different numbers instead of always passing 2? also for other tests below

Reply:
The number `2` is the number of assertions that QUnit should expect. Initially I did that using `expect(2)` but because Apps Script Editor shows warning as the preferred syntax is passing this as a parameter to `QUnit.test()`, I refactored it to this.